From 2feaffe075716b9398bf045924094a0a1b5b57aa Mon Sep 17 00:00:00 2001
From: Subbaraya Sundeep <sbhatta@marvell.com>
Date: Mon, 19 Aug 2019 12:01:27 +0530
Subject: [PATCH 319/767] octeontx2-af: Change message level to debug

commit 1a16c0f852a40bba93a6f3170992fba342ca811f from
git@git.assembla.com:cavium/WindRiver.linux.git

Message responses may time out at sender if there
is no receipent hence make the time out message as
dev_dbg.

Change-Id: Id68d7294f2ebe52d5530692703da2bb7c44c825a
Signed-off-by: Subbaraya Sundeep <sbhatta@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/14491
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/mbox.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/mbox.c b/drivers/net/ethernet/marvell/octeontx2/af/mbox.c
index 75dbcacc23c9..1a245e15e751 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/mbox.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/mbox.c
@@ -144,7 +144,7 @@ int otx2_mbox_wait_for_rsp(struct otx2_mbox *mbox, int devid)
 		msleep(sleep);
 		timeout += sleep;
 		if (timeout >= MBOX_RSP_TIMEOUT) {
-			dev_err(sender, "timed out while waiting for rsp\n");
+			dev_dbg(sender, "timed out while waiting for rsp\n");
 			return -EIO;
 		}
 	}
-- 
2.31.1

